Add CLI Examples for Application Insights (#9006)
aws/aws-cli: awscli/examples/application-insights/create-application.rst

**To create an application from a resource group**

> This example uses the ``create-application command`` to create an application from a resource group. ::

Reviewer: ```suggestion
The following ``create-application`` creates an application from a resource group. ::
```
A few of your examples have a typo where the word "command" was in the code formatting. When fixing the formatting in all your intro lines, please check for this fix as well.
